OWNERS

```diff
@@ -0,0 +1,24 @@
+allowed_users:
```

Comment:
Just out of curiosity, what is `allowed_users` in the OWNERS file?

Reply:
@tisutisu any chance you know what it is? If not, I'd say let's just drop it

Reply:
I think `allowed_users` is a leftover in the OWNERS file now.. it is being used [here](https://github.com/konflux-ci/build-tasks-dockerfiles/blob/main/.github/workflows/check-author-in-owner-file.yml) ... workflow itself does not serve any purpose now .. IIRC we attempted something in the workflow so that only `allowed_users` can do some intended action.. don't remember exactly what was it

Reply:
Do we require to set up the mentioned workflow for this repo?

Reply:
Remove  with [this PR](https://github.com/konflux-ci/build-tasks-dockerfiles/pull/192)

Reply:
`allowed_users` is removed from the `OWNERS` file.